process/graduation_criteria.md

```diff
@@ -14,6 +14,7 @@ To be accepted in the sandbox a project must
 * Adopt the CNCF [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
 * Adhere to CNCF [IP Policy](https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy) (including trademark transferred)
 * List their sandbox status prominently on website/readme
+* Explicitly define a project governance and committer process. The committer process should cover the full committer lifecycle including onboarding and offboarding or emeritus criteria. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md or MAINTAINERS.md file showing the current and emeritus committers. See https://contribute.cncf.io/maintainers/governance/ for more info.
```

Comment:
I'm also going to suggest that an OWNERS file is not a substitute for MAINTAINERS.md.  This is because the owners file contains ONLY github handles, and not contact information, affiliation, full name, or any other information about the maintainer.  If the goal is to make the project more transparent to the TOC and potential contributors, then we need a full maintainers file.

Reply:
Although: the combination of a README with the extra information together with an OWNERS file works.  It might be better to define what information needs to be exposed about maintainers.

Reply:
It *is* currently defined: it needs to show who the current and emeritus committers are. We don't need to determine exactly what is and isn't required or how the information is presented - when assessing a project, the TOC can simply look at the file and consider whether information about committers is present

This drive towards consistency across projects feels like a slippery slope towards yet more make-work process requiring busy project maintainers to re-write files to fit some arbitrary format. How does consistency between projects really help people who want to contribute to a project? Most folks concentrate on one thing at a time.